testonly/mapcontents.go

```diff
@@ -127,6 +129,114 @@ func (m *MapContents) UpdatedWith(rev uint64, leaves []*trillian.MapLeaf) *MapCo
 	return &result
 }
 
+type bitString struct {
```

Comment:
`big.Int` _might_ do a lot of what you want from this?

Reply:
It would do single-bit setting, but would make clearing suffixes harder and is also a bit misleading (this value is never interpreted as an integer) so I'd prefer to stick as-is.